# This represents a pose in free space with uncertainty.

geometry_msgs/Pose pose

# float weight
float32 weight
